rust/lance/src/dataset/scanner.rs

```diff
@@ -852,24 +852,24 @@ impl Scanner {
         } else {
             Arc::new(vec![])
         };
-        let knn_idx = indices.iter().find(|i| i.fields.contains(&column_id));
-        if let Some(index) = knn_idx {
+        if let Some(index) = indices.iter().find(|i| i.fields.contains(&column_id)) {
```

Comment:
nit: maybe we should save this in manifest

Reply:
what is `this` you mentioned to?

Reply:
oh as in `column_id` -> index mapping